README.md

```diff
@@ -125,8 +125,13 @@ read:
 write:
   pragmas: <list of pragma settings for read/write>
 ```
-Please refer to [documentation of pragmas](https://www.sqlite.org/pragma.html).
+
+By default, SQLite settings are significantly optimized for performance.
+This might have consequences of bag data being corrupted after an application or system-level crash.
```

Comment:
It would be good to clarify that only the current bagfile has the possibility to be corrupted in this case, any already-split files are safe once finalized, so the maximum data loss is bounded by the maximum bag (time or size)